packages/fxa-admin-server/src/scripts/audit-tokens.ts

```diff
@@ -69,7 +69,7 @@ const tables = {
   passwordForgotTokens: toTable('passwordForgotTokens'),
   paypalCustomers: toTable('paypalCustomers'),
   recoveryCodes: toTable('recoveryCodes'),
-  recoveryKeys: toTable('recoveryCodes'),
+  recoveryKeys: toTable('recoveryKeys'),
```

Comment:
@dschom Spotted that this had `recoveryCodes` instead of `recoveryKeys`. Was that a typo, or was that intentional?

Reply:
A typo! Thank you for catching this!